config/locales/exceptions.en.yml

```diff
@@ -2339,3 +2339,12 @@ en:
         detail: Order unable to be processed
         code: MDOT_unprocessable_entity
         status: 422
+
+  chip:
+    exceptions:
+      unmapped_service_exception:
+        <<: *defaults
+        title: 'Operation failed'
+        detail: The upstream server returned an error code that is unmapped
+        code: unmapped_service_exception
+        status: 400
```

Comment:
Is it possible to have this defined in a separate `exceptions.en.yml` file specific to chip lib?

Reply:
Discussed offline; lib seems to be referring to `config/locales/exceptions.en.yml`